src_c/draw.c

```diff
@@ -102,10 +102,6 @@ aaline(PyObject *self, PyObject *arg)
         return NULL;
     surf = pgSurface_AsSurface(surfobj);
 
-    if (surf->format->BytesPerPixel != 3 && surf->format->BytesPerPixel != 4)
```

Comment:
oh cool. I guess this means it supports more than 32/24 bit now.

Reply:
I think it's always between 1 and 4, but 2 and 1 also work now